sos/report/plugins/ovn_central.py

```diff
@@ -145,15 +145,20 @@ def setup(self):
         self.add_copy_spec("/etc/sysconfig/ovn-northd")
 
         ovs_dbdir = os.environ.get('OVS_DBDIR')
-        for dbfile in ['ovnnb_db.db', 'ovnsb_db.db']:
-            self.add_copy_spec([
-                self.path_join('/var/lib/openvswitch/ovn', dbfile),
-                self.path_join('/usr/local/etc/openvswitch', dbfile),
-                self.path_join('/etc/openvswitch', dbfile),
-                self.path_join('/var/lib/openvswitch', dbfile),
-                self.path_join('/var/lib/ovn/etc', dbfile),
-                self.path_join('/var/lib/ovn', dbfile)
-            ])
+        for dbfile in ["ovnnb_db.db", "ovnsb_db.db"]:
+            for path in [
+                "/var/lib/openvswitch/ovn",
+                "/usr/local/etc/openvswitch",
+                "/etc/openvswitch",
+                "/var/lib/openvswitch",
+                "/var/lib/ovn/etc",
+                "/var/lib/ovn",
+            ]:
+                dbfilepath = self.path_join(path, dbfile)
+                if os.path.exists(dbfilepath):
+                    self.add_copy_spec(dbfilepath)
+                    self.add_cmd_output(
+                        "ls -lan %s" % dbfilepath, foreground=True)
```

Comment:
The `os.path.exists()` check is superfluous. `add_copy_spec()` gracefully handles when a provided file path does not exist, and we won't capture output from any failed `ls` commands for the same reason.

Reply:
Yeah, but without that os.path.exists(), we can trust on add_copy_spect() to get the file copy, but how we can jump the add_cmd_output("ls -lan...") for files that dont exist without that check? because if we run add_cmd_output("ls -lan...") over an non-existing file I already checked that a output file was written to sosreport with a "file not found". Or maybe I missunderstand something?

Reply:
Ah, I thought `ls` gave a different error code than it does in that scenario, my mistake.

Reply:
Ack, this is fine.